src/main/resources/assets/wurst/lang/ja_jp.json

```diff
@@ -0,0 +1,139 @@
+    "description.wurst.hack.nooverlay": "水と溶岩のオーバーレイを表示しません。",
+    "description.wurst.hack.nopumpkin": "頭にカボチャをかぶったときにオーバーレイを表示しません",
+    "description.wurst.hack.noslowdown": "ハチミツ、ソウルサンド、アイテムの使用によるスロー効果を解除します。",
+    "description.wurst.hack.noweather": "クライアント側の天気、時間、月の満ち欠けを変更できます。",
+    "description.wurst.hack.noweb": "蜘蛛の巣によって遅くなるのを防ぎます。",
+    "description.wurst.hack.nuker": "あなたの周りのブロックを自動的に壊します。",
+    "description.wurst.hack.nukerlegit": "すべてのアンチチートプラグインをバイパスする遅いNuker。\n通常のNoCheat +サーバーでは必要ありません！",
+    "description.wurst.hack.openwateresp": "「オープンウォーター」で釣りをしているかどうかを示し、オープンウォーターの計算に使用される領域の周りにボックスを描画します。",
+    "description.wurst.hack.overlay": "ブロックをマイニングするたびにNukerアニメーションをレンダリングします。",
+    "description.wurst.hack.panic": "有効なすべてのハックを即座にオフにします。\nこれに注意してください！",
+    "description.wurst.hack.parkour": "ブロックの端に到達したときに自動的にジャンプします。\nパルクールやジャンプ'n'ランに便利です。",
+    "description.wurst.hack.playeresp": "近くのプレイヤーをハイライトします。\n友達のESPボックスは青色で表示されます。",
+    "description.wurst.hack.playerfinder": "雷雨の際に遠くのプレイヤーを見つけます。",
+    "description.wurst.hack.portalgui": "ポータルでGUIを開くことができます。",
+    "description.wurst.hack.potionsaver": "あなたが静止している間、すべてのポーション効果を凍結します。",
+    "description.wurst.hack.prophuntesp": "Prophuntで偽のブロックを表示できます。\nMineplexProphuntのために作られました。 他のサーバーでは動作しない可能性があります。",
+    "description.wurst.hack.protect": "最も近いエンティティを追跡し、他のエンティティから保護するボット。\nUse .protectは、最も近いエンティティではなく特定のエンティティを保護します。",
+    "description.wurst.hack.radar": "近くのエンティティの場所を表示します。\n§cred§r - プレイヤー\n§6orange§r - モンスター\n§agreen§r - 動物\n§7gray§r - その他",
+    "description.wurst.hack.rainbowui": "§cM§aa§9k§ce§as §9e§cv§ae§9r§cy§at§9h§ci§an§9g §cc§ao§9l§co§ar§9f§cu§al§9.",
+    "description.wurst.hack.reach": "より遠くへの到達を可能にします。",
+    "description.wurst.hack.remoteview": "世界を他の誰かとして見ることができます。\n.rvコマンドを使用して、特定のエンティティをターゲットにします。",
+    "description.wurst.hack.safewalk": "ふちから落ちるのを防ぎます。",
+    "description.wurst.hack.scaffoldwalk": "足元にブロックを自動的に配置します。",
+    "description.wurst.hack.search": "特定のブロックをレインボーカラーでハイライトすることで、ブロックの検索をサポートします。",
+    "description.wurst.hack.servercrasher": "1.15.xサーバーをクラッシュさせる可能性のあるアイテムを生成します。\n§oクリエイティブモードのみ。§r",
+    "description.wurst.hack.skinderp": "skinの一部がランダムに切り替わります。",
+    "description.wurst.hack.sneak": "自動的にしゃがみます。",
+    "description.wurst.hack.snowshoe": "パウダースノーの上を歩けるようになります。",
+    "description.wurst.hack.speedhack": "スプリントやジャンプと比べて2.5倍の速さで走ることができます。\n\n§6§l警告:§r NoCheat+のバージョン3.13.2でパッチが当てられました。古いバージョンのNoCheat+をバイパスするだけです。\nType §l/ncp version§r を入力すると、サーバーのNoCheat+バージョンを確認することができます。",
+    "description.wurst.hack.speednuker": "NoCheat+を回避できないNukerの高速バージョン。",
+    "description.wurst.hack.spider": "蜘蛛のように壁を登れるようになります。",
+    "description.wurst.hack.step": "フルブロックのステップアップを可能にします。",
+    "description.wurst.hack.throw": "1つのアイテムを複数回使用します。雪玉や卵を投げたり、モブを出現させたり、トロッコを置いたりと、非常に大量に使用することができます。\n\nこれが原因でラグが発生し、サーバーがクラッシュすることもあります。",
+    "description.wurst.hack.tillaura": "土や草などを自動的に農地にします。\nKillauraと一緒に使用してはいけません。",

```

Comment:
Does this say "Do not use with Killaura"? It should say "Do not _confuse_ with Killaura" instead.